plugins/ibm_qradar/plugin.spec.yaml

```diff
@@ -0,0 +1,982 @@
+      filter:
+        type: string
+        required: false
+        description: This parameter is used to restrict the elements in a list base on the contents of various fields
+        example: id=1000
+      fields:
+        type: string
+        required: false
+        description: Use this parameter to specify which fields you would like to get back in the response. Fields that are not named are excluded. Specify subfields in brackets and multiple fields in the same object are separated by commas
+        example: id
+      include_deleted:
+        type: boolean
+        required: false
+        description: If true, deleted closing reasons are included in the response. Defaults to false. Deleted closing reasons cannot be used to close an offense
+        example: false
+      include_reserved:
+        type: boolean
+        required: false
+        description: If true, reserved closing reasons are included in the response. Defaults to false. Reserved closing reasons cannot be used to close an offense
+        example: false
+    output:
+      data:
+        title: Offense Closing Reasons
+        type: "[]closing_reason"
+        description: JSON Data of the Offense Closing Reasons
+        required: false
+  update_offense:
+    title: Update Offenses
+    description: Update Offenses
+    input:
+      offense_id:
+        type: integer
+        required: true
+        description: The ID of the offense to update
+        example: 100
+      assigned_to:
+        type: string
+        required: false
+        description: A user to assign the offense to
+        example: admin
+      closing_reason_id:
+        type: string
+        required: false
+        description: The ID of a closing reason. You must provide a valid closing_reason_id when you close an offense
+        example: 100
+      fields:
+        type: string
+        required: false
+        description: Use this parameter to specify which fields you would like to get back in the response. Fields that are not named are excluded. Specify subfields in brackets and multiple fields in the same object are separated by commas
+        example: id
+      follow_up:
+        type: boolean
+        required: false
+        description: Set to true to set the follow up flag on the offense
+        example: false
+      protected:
+        type: boolean
+        required: false
+        description: The ID of a closing reason. You must provide a valid closing_reason_id when you close an offense
+        example: false
+      status:
+        type: string
+        required: false
+        description: The new status for the offense. Set to one of the OPEN, HIDDEN, CLOSED. When the status of an offense is being set to CLOSED, a valid closing_reason_id must be provided. To hide an offense, use the HIDDEN status. To show a previously hidden offense, use the OPEN status
+        enum:
+          - "OPEN"
+          - "HIDDEN"
+          - "CLOSED"
+          - ""
```

Comment:
```suggestion
          - "Open"
          - "Hidden"
          - "Closed"
          - ""
```

Reply:
THis is expected by Qradar API so we have to keep as it is.

Reply:
@hiren-sacumen It should use proper capitalization as in the suggestion I left. You should adjust it to what the API requires in your Python code. The inputs given in actions/triggers do not need to be 1:1 with what the API requires since the plugin is just an easier-to-use abstraction over the API.